### PR TITLE
[fix](orderby)disallow hll and bitmap data type in order by list

### PR DESCRIPTION
**Pull request by starocean999**
# Proposed changes

Issue Number: close #xxx

## Problem summary

disallow hll and bitmap data type in order by list

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/query/aggregate/aggregate_group_by_hll_and_bitmap.groovy [hunk 1]
```diff
@@ -35,5 +35,15 @@ suite("aggregate_group_by_hll_and_bitmap") {
         exception "Doris hll and bitmap column must use with specific function, and don't support filter or group by.please run 'help hll' or 'help bitmap' in your mysql client"
     }
 
+    test {
+        sql "select user_ids from test_group_by_hll_and_bitmap group by user_ids"
+        exception "Doris hll and bitmap column must use with specific function, and don't support filter or group by.please run 'help hll' or 'help bitmap' in your mysql client"
+    }
+
+    test {
+        sql "select hll_set from test_group_by_hll_and_bitmap group by hll_set"
```
Comment (spaces-X) [inline regression-test/suites/query/aggregate/aggregate_group_by_hll_and_bitmap.groovy:44]:
```suggestion
        sql "select hll_set from test_group_by_hll_and_bitmap order by hll_set"
```

File: regression-test/suites/query/aggregate/aggregate_group_by_hll_and_bitmap.groovy [hunk 2]
```diff
@@ -35,5 +35,15 @@ suite("aggregate_group_by_hll_and_bitmap") {
         exception "Doris hll and bitmap column must use with specific function, and don't support filter or group by.please run 'help hll' or 'help bitmap' in your mysql client"
     }
 
+    test {
+        sql "select user_ids from test_group_by_hll_and_bitmap group by user_ids"
```
Comment (spaces-X) [inline regression-test/suites/query/aggregate/aggregate_group_by_hll_and_bitmap.groovy:39]:
```suggestion
        sql "select user_ids from test_group_by_hll_and_bitmap order by user_ids"
```

Comment (starocean999) [reply regression-test/suites/query/aggregate/aggregate_group_by_hll_and_bitmap.groovy]: Done, thanks.

File: regression-test/suites/query/aggregate/aggregate_group_by_hll_and_bitmap.groovy (hunk 1 above)
Comment (starocean999) [reply regression-test/suites/query/aggregate/aggregate_group_by_hll_and_bitmap.groovy]: Done, thanks.

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**